src/dashboard/src/requirements/base.txt

```diff
@@ -39,16 +39,18 @@ lxml==3.5.0               # via -r base.in, metsrw
 metsrw==0.3.15            # via -r base.in
 mysqlclient==1.4.6        # via -r base.in, agentarchives
 ndg-httpsclient==0.5.1    # via -r base.in
+numpy==1.16.6             # via pandas
+pandas==0.24.2            # via -r base.in
```

Comment:
Please don't see it as an objection but as a curiosity - the size of numpy + pandas together is near the size of the rest of our dependencies in dashboard, see https://gist.github.com/sevein/43bb237f7139e22c9f16e5bfddb69295.